executor.go

```diff
@@ -350,7 +351,16 @@ func (e *Executor) launchTask(taskInfo mesos.TaskInfo) (Command, error) {
 		if err != nil {
 			return nil, fmt.Errorf("cannot configure service log scraping: %s", err)
 		}
-		cmdOption = ScrapCmdOutput(scr, apr)
+		extenders := []servicelog.Extender{
```

Comment:
This is becoming quite a lot of code in the executor package, can we make it just one line and do all the filters, extending etc. in the servicelog package or something similar?

Reply:
I can move it to separate function. Will it be okay?

Reply:
Well, that will make it more readable. If you don't want to do it in a package than I can live with a function at least.

Reply:
I cannot do this in `servicelog` package because it will add circular dependency between `executor` and `servicelog` packages.

Reply:
@AlfredBroda fixed.